src/main/resources/application.properties

```diff
@@ -3,4 +3,7 @@ spring.data.mongodb.auto-index-creation=true
 server.port=${SERVER_PORT:8081}
 spring.kafka.consumer.group-id=astha
 auth.token.public.key=${PUBLIC_KEY}
-spring.kafka.bootstrap-servers=${KAFKA_BOOTSTRAP_SERVERS}
\ No newline at end of file
+spring.kafka.bootstrap-servers=${KAFKA_BOOTSTRAP_SERVERS}
+
+springdoc.api-docs.path=/api-docs
+springdoc.swagger-ui.path=/swagger-ui-custom.html
```

Comment:
nit: Put a blank line at the end of file

Reply:
fixed